contracts/FullCheckpoint.sol

```diff
@@ -279,16 +275,20 @@ contract FullCheckpoint {
     }
 
     function setCommittedStatus(bytes32 startBlock) internal {
```

Comment:
Is committed and finalized same? If so, please add a comment for this function

Reply:
finalized==false equals mainnetNum==-1